### DISCUSSION TITLE
Second Profile add in contributors in first Profile

### DISCUSSION THREAD

**Discussion by khe4oyan** [community/community#65168] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello. For a long time I can not fix one problem.

Earlier, I added my second github account to git in order to create repositories from this account and work on them, but now I don’t need it anymore, but I can’t remove the second account from the terminal in any way. Each "push" is done on behalf of the second account.

I need to completely remove it in order to push on behalf of my main account.

Note: The second account is permanently added as "contributors" to the main account.

Who can help me with this problem?

**Comment by arafajs** [comment, marked as answer]
**1. Remove the remote URL for the second account:**
`git remote remove origin`
**2. Remove the credentials for the second account:**
`git credential-manager reject https://github.com`
**3. Re-add the remote URL for your main account:**
`git remote add origin https://github.com/YOUR_USERNAME/YOUR_REPO.git`
**4. Verify that the origin remote points to your main account:**
`git remote -v`
**5. Set the username and email to use your main account details:**
```
git config user.name "YOUR_USERNAME"
git config user.email "your_email@example.com"
```
**6.  Push to origin to make sure you are pushing with your main account:**
`git push origin master`

**Reply by khe4oyan** [reply]
Thank you!

**Comment by 0dayhunter** [comment]
It sounds like you're trying to remove a secondary GitHub account from your local Git configuration and have your commits associated with your primary account instead. If the secondary account is set as a contributor in your repositories, it shouldn't directly affect the way you push commits from your local machine.

Here's a step-by-step guide to help you resolve this issue:

**Step 1: Check Global Configuration**
1. Open your terminal.
2. Run the following command to view your global Git configuration:
   ```
   git config --global --list
   ```
3. Look for entries related to the second GitHub account, such as `user.name` and `user.email`. These settings are what Git uses to identify you when you commit.

**Step 2: Update Global Configuration**
If you find that your global Git configuration is set to the second account, you can update it to match your primary GitHub account:
1. Run the following commands to set your name and email for your primary GitHub account:
   ```
   git config --global user.name "Your Primary Name"
   git config --global user.email "your.primary@email.com"
   ```

**Step 3: Check Repository-specific Configuration**
Sometimes, repositories have their own configurations that override the global settings. Make sure your repositories aren't set up to use the second account's credentials:
1. Navigate to the repository in your terminal:
   ```
   cd /path/to/your/repository
   ```
2. Run the following command to view the repository-specific configuration:
   ```
   git config --local --list
   ```
3. Look for any entries related to the second GitHub account and update them if necessary.

**Step 4: Check Remote URLs**
Make sure that the remote URLs for your repositories are configured to use your primary GitHub account:
1. Run the following command to view the remote URLs for each repository:
   ```
   git remote -v
   ```
2. If you see URLs with the second account, update them using the following command:
   ```
   git remote set-url origin <primary_account_url>
   ```

**Step 5: Commit and Push**
After you've updated your configuration, commit and push a change to one of your repositories to see if the changes take effect.

Remember that Git tracks your identity on a per-repository basis, so each repository can have different user information set. Make sure you're updating both global and repository-specific configurations if needed.

If you continue to face issues, it might help to provide more specific details about the commands you're using and any error messages you're encountering.
